app/lib/three_scale/address.rb

```diff
@@ -57,7 +57,7 @@ def to_hash
   end
 
   # Quacking like Hash makes BillingAddress compatible with
-  # ActiveMerchant. See AuthorizeNetCimGateway#add_address for interface.
+  # ActiveMerchant.
```

Comment:
Not sure it's an example, why mention AuthorizeNet specifically? I think it referst to this https://github.com/activemerchant/active_merchant/blob/ebcb303d507ea90ac9deddbdf2622a8fd1dfbb46/lib/active_merchant/billing/gateways/authorize_net_cim.rb#L777

I don't understand the comment, TBH. This was [added 11 years ago](https://github.com/3scale/system/commit/07d71f9e134ed5a94aaa12f7b792f9d05ef5f2b2#diff-d7d2bfdc76fca3c310d1cf1450a1893de516ae08dc151c4369b4e484bad9f1cbR41) and maybe it is poorly written. Maybe it's not needed anymore. Why mention AuthorizeNet specifically???? 🤯

Reply:
You are right. I think it was pointing there as documentation of what should be supported/expected. So we still need such information. Otherwise there is no clue of what this method should support.

I'd say that either update with a better soure of information or keep the existing comment because it at least provides some clues for a future reader.